packages-backend/loggers/src/formatters/rewrite-fields.ts

```diff
@@ -9,16 +9,31 @@ type TRewriteField = {
   from: string;
   to: string;
   replaceValue?: (value: unknown) => unknown;
+  unsetFromField?: boolean;
```

Comment:
Argued with myself for a name. This is the best I could come up with.

Reply:
Would it be better to have it `false` by default and enable the functionality with `true`? That's what we usually do with component props, right?

```suggestion
  keepOriginalField?: boolean;
```

Reply:
Works for me too. Was an alternative I pondered with. I liked the term unset as it follows the implementation nicely. 

keep* or preserve* are the inverse options. I can change it.